Configuration.props

```
@@ -145,6 +145,7 @@
     <MonoSourceFullPath>$([System.IO.Path]::GetFullPath ('$(MonoSourceDirectory)'))</MonoSourceFullPath>
     <SqliteSourceFullPath>$([System.IO.Path]::GetFullPath ('$(SqliteSourceDirectory)'))</SqliteSourceFullPath>
     <OpenTKSourceFullPath>$([System.IO.Path]::GetFullPath ('$(OpenTKSourceDirectory)'))</OpenTKSourceFullPath>
+    <JavaInteropTargetFrameworkVersion>net8.0</JavaInteropTargetFrameworkVersion>
```

Should this be:
```suggestion
    <JavaInteropTargetFrameworkVersion>$(DotNetStableTargetFramework)</JavaInteropTargetFrameworkVersion>
```
Or will we likely bump this independently of moving to .NET 10, for example?

---

`$(JavaInteropTargetFrameworkVersion)` is *reactionary*; the Java.Interop repo targets .NET versions on its own ("lazy") schedule, and the entire reason it's on .NET 8 *now* is because I want to start using NativeAOT things in there (xamarin/java.interop#1153); if not for that, it would still be on .NET *7*, with no real need or reason to update it otherwise.

Thus, it should *not* be `$(DotNetStableTargetFramework)`, because that would require that Java.Interop bump to the latest .NET version far faster than it has historically.

(For comparison, Java.Interop did not target .NET 7 until 2022-May, ~7 months after .NET 7 went stable…)